docs/index.md

```diff
@@ -2,35 +2,33 @@
 sidebar_position: 1
 ---
 
-# Introduction
+# Welcome to DBOS!
 
-Welcome to DBOS!
+DBOS Transact is an open source framework for developing database-backed applications in Typescript and PostgreSQL.
+You can host DBOS Transact applications yourself or deploy them in minutes to DBOS Cloud.
 
-### What is DBOS?
+You want to build your next database-backed application with DBOS Transact because:
 
-DBOS is a transactional serverless framework and platform that helps you develop and deploy database-backed applications.
-You develop your applications in TypeScript and PostgreSQL with our [open-source SDK](https://github.com/dbos-inc/dbos-ts) then deploy them in minutes to DBOS Cloud.
-
-You want to build your next database-backed application with DBOS because:
-
-- **It's simple**.  Write your business logic using serverless functions, run them locally, and deploy them to the cloud in minutes.  Store all your data in Postgres&#8212;we'll manage the connections and transactions for you.
+- **It's simple**.  Write your business logic with serverless functions and either [self-host them](./tutorials/self-hosting.md) or deploy them to the cloud in minutes.  Store all your data in Postgres&#8212;we'll manage the connections and transactions for you.
```

Comment:
You don't want to put a second link over "deploy them to the cloud" -> cloud quickstart? I thought that might look good